config/envs/default.py

```diff
@@ -98,15 +98,15 @@ class DefaultConfig:
         "/assessments/get-stats/{fund_id}/{round_id}?{params}"
     )
 
-    ASSESSMENTS_STATS_FLAGS_V2_ENDPOINT = (
-        "/assessments/get-stats_flags_v2/{fund_id}/{round_id}?{params}"
+    ASSESSMENTS_STATS_FLAGS_ENDPOINT = (
```

Comment:
looks like you missed refactoring this....it's duplicated on line 98

Reply:
good spot

Reply:
can we name it `APPLICATION_OVERVIEW_ENDPOINT_FUND_ROUND_PARAMS` like before ? this is not a flag specific endpoint right